src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.cs

```diff
@@ -1848,7 +1851,8 @@ private bool ConsumeMultiLineComment(ReadOnlySpan<byte> localBuffer, int previou
                 return false;
             }
 
-            ValueSpan = _buffer.Slice(previousConsumed, idx + 3); // Include the slash/asterisk and final slash at the end of the comment as part of it.
+            // Exclude /* and trailing */ in the comment
+            ValueSpan = _buffer.Slice(previousConsumed + 2, idx);
```

Comment:
Does this need to be idx - 1 to match the multi-segment case, here: https://github.com/dotnet/corefx/pull/35934/files#diff-ed80dc6b290b6833c7ed8386409ca144R2523:
```C#
                // Exclude the /* and the trailing *
                ValueSpan = _buffer.Slice(previousConsumed + 2, i - 1);
```

Reply:
This is being reworked along with the changes to use sequence positions in correct way.